.github/workflows/all_solutions.yml

```diff
@@ -2,15 +2,10 @@ name: .NET Agent All Solutions Build
 
 # Does not run on PUSH since we have already ran all the test
 on:
-  push:
-    branches:
-    # Forgive me, fellow engineers
-      - jcoleman/**
   pull_request:
     branches: 
     - main
-    # TODO: Trim everything below this when appropriate
-    - nservicebus7-feature-branch
+    - '**-feature-branch'
```

Comment:
not sure the quotes are needed

Reply:
Without the quotes it doesn't look right in vscode in terms of syntax highlighting; also, one of the examples I found in GHA documentation had the quotes.